docs/src/tutorials/low_level.md

```diff
@@ -1,4 +1,4 @@
-# Investigating `symbolic_discretize` with the 1-D Burgers' Equation
+# Investigating `symbolic_discretize` with the `PhysicsInformedNN` Discretizer for the 1-D Burgers' Equation
```

Comment:
That's kind of a given since non-Bayesian is standard, I don't think the extra phrase is needed.
```suggestion
# Investigating `symbolic_discretize` for the 1-D Burgers' Equation
```